### PR TITLE
Add a clarification on Reonboarding and Router Devices

### PR DESCRIPTION
**Pull request by Frank-Wiebeler**
When a router device is used, Reonboarding is opsolete. Instead, a test for router devices is advised

### CODE REVIEW

File: modules/ROOT/pages/certification.adoc [hunk 1]
```diff
@@ -152,6 +152,8 @@ a|
 
 |xref:./integration/reonboarding.adoc[Re-onboarding]
 |Always
+
+(if no RouterDevice is used)
```
Comment (saschadoemer) [inline modules/ROOT/pages/certification.adoc:156]:
```suggestion
(if the application does not use router devices)
```

File: modules/ROOT/pages/certification.adoc [hunk 2]
```diff
@@ -161,6 +163,17 @@ a|
 ** Wrong account: During re-onboarding, the user is logged in with a different agrirouter account than before. This should result in a new endpoint onboarding in a different account.
 
 
+|xref:./router-devices.adoc[Updating RouterDevice]
+| Farming Software
+
+Telemetry Platforms
+
+(If RouterDevice is used)
```
Comment (saschadoemer) [inline modules/ROOT/pages/certification.adoc:171]:
```suggestion
(If the application uses router devices)
```

File: modules/ROOT/pages/certification.adoc [hunk 3]
```diff
@@ -161,6 +163,17 @@ a|
...
+a|
+* The app provider needs to demonstrate that he is able to replace the RouterDevice with a new one and that the communication via this new RouterDevice can be continued.
```
Comment (saschadoemer) [inline modules/ROOT/pages/certification.adoc:173]:
```suggestion
* The app provider has to demonstrate that he is able to replace the router device with a new one and that the communication via this new router device can be continued.
```

**Review by saschadoemer — COMMENTED** [review]
(☞ ͡° ͜ʖ ͡°)☞

**Review by oliverrahner — APPROVED** [review]

**Pull request merged**